detection-rules/impersonation_netflix.yml

```
@@ -0,0 +1,19 @@
+name: "Brand impersonation: Netflix"
+description: |
+  Impersonation of Netflix.
+references:
+  - "https://news.trendmicro.com/2023/01/18/netflix-scams-2023-job-text-email/"
+type: "rule"
+severity: "low"
+source: |
+  type.inbound
+  and (
+      strings.ilike(sender.display_name, '*netflix*')
+      or strings.ilevenshtein(sender.display_name, 'netflix') <= 1
+      or strings.ilike(sender.email.domain.domain, '*netflix*')
+  )
+  and sender.email.domain.domain !~ 'netflix.com'
```

```suggestion
  and sender.email.domain.root_domain !~ 'netflix.com'
```

This was firing on message samples sent from mailer.netflix.com. 
The sender.email.email value was "mailer.netflix.com" 
Using sender.email.domain.root_domain captures your intent I believe.

---

Yes, sender.email.domain.root_domain was my intent. Thank you.